rotkehlchen/accounting/constants.py

```diff
@@ -319,6 +321,9 @@
     )}, EventCategory.DELEGATE: {DEFAULT: EventCategoryDetails(
         label='delegate',
         icon='lu-handshake',
+    )}, EventCategory.LOSS: {DEFAULT: EventCategoryDetails(
```

Comment:
I would like you to create a specific event category for liquidity provision loss. As icon perhaps use the droplet + trending down for LP loss? And for generic loss what you used. I think @lukicenturi knows how to combine them.